### PR TITLE
[docs] - Add Cloud PagerDuty integration (DOC-161)

### PR DESCRIPTION
**Pull request by erinkcochran87**
## Summary & Motivation

This PR adds a guide for the new PagerDuty + Dagster Cloud integration.

## How I Tested These Changes

:eyes:

### CODE REVIEW

File: docs/content/dagster-cloud/managing-deployments/alerts/pagerduty.mdx [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+
+   - **If creating a new service for your integration**: Refer to PagerDuty's [Configuring services and integrations documentation](https://support.pagerduty.com/docs/services-and-integrations#section-configuring-services-and-integrations) and follow the steps outlined in the [Create a new service](https://support.pagerduty.com/docs/services-and-integrations#section-create-a-new-service) section, selecting **Dagster Cloud** as the **Integration Type** in step 4. Continue with the [**In Dagster Cloud**](#in-dagster-cloud) section (below) once finished.
+
+3. Enter an **Integration Name** in the format `monitoring-tool-service-name` (e.g. `dagster-cloud-run-failures`) and select **Dagster Cloud** from the **Integration Type** menu.
+
+4. Click the **Add Integration** button to save your new integration. You will be redirected to the **Integrations** tab for your service.
+
+5. An **Integration Key** will be generated on this screen. Keep this key saved in a safe place, as it will be used when you configure the integration with **Dagster Cloud** in the next section.
+
+   ![](https://pdpartner.s3.amazonaws.com/ig-template-copy-integration-key.png)
```
Comment (erinkcochran87) [inline docs/content/dagster-cloud/managing-deployments/alerts/pagerduty.mdx:66]: I'm going to replace this - just wanted to get the draft in front of you more quickly.

File: docs/content/dagster-cloud/managing-deployments/alerts/pagerduty.mdx [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+
+   1. Select **PagerDuty**.
+   2. In the field that displays, paste the PagerDuty integration key you created in the previous section.
+
+4. Click **Save policy**.
+
+</TabItem>
+<TabItem name="dagster-cloud CLI">
+
+### dagster-cloud CLI
```
Comment (erinkcochran87) [inline docs/content/dagster-cloud/managing-deployments/alerts/pagerduty.mdx:91]: I assumed PagerDuty follows the same pattern as the other integrations, but let me know if this is incorrect or not supported

File: docs/content/dagster-cloud/managing-deployments/alerts/pagerduty.mdx [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+  - name: "pagerduty-alert-policy"
+    description: "An alert policy to create a PagerDuty incident upon job failure."
+    tags:
+      - key: "level"
+        value: "critical"
+    event_types:
+      - "AGENT_UNAVAILABLE"
+    notification_service:
+      pagerduty:
+        integration_key: TODO: USE ENVVAR?
```
Comment (erinkcochran87) [inline docs/content/dagster-cloud/managing-deployments/alerts/pagerduty.mdx:108]: I assume we'd want an env var here - what would this look like in this file?

File: docs/content/_navigation.json [hunk 4]
```diff
@@ -609,6 +609,10 @@
                     "title": "Microsoft Teams",
                     "path": "/dagster-cloud/managing-deployments/alerts/microsoft-teams"
                   },
+                  {
+                    "title": "PagerDuty",
```
Comment (johannkm) [inline docs/content/_navigation.json:613]: We shouldn't list this page until the integration is available, so we can either
- remove it from the nav so it's unlisted, but we can land this diff
- wait to land once the integration is ready, and use the vercel link for the pagerduty application

up to you

File: docs/content/dagster-cloud/managing-deployments/alerts/pagerduty.mdx [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+- Incidents can be triggered for events related to runs, assets, schedules, sensors, and code locations
+- Users must resolve incidents in PagerDuty. Dagster Cloud doesn't currently automatically resolve incidents.
+
+### Requirements
+
+To integrate PagerDuty with Dagster Cloud, you'll need:
+
+- **Permissions in PagerDuty that allow you to manage services and integrations.** Refer to PagerDuty's [User roles](https://support.pagerduty.com/docs/user-roles) documentation for more information.
+- **The following in Dagster Cloud**:
+  - **A Pro plan**
```
Comment (johannkm) [inline docs/content/dagster-cloud/managing-deployments/alerts/pagerduty.mdx:39]: I think we could add a callout for this

File: docs/content/dagster-cloud/managing-deployments/alerts/pagerduty.mdx [hunk 6]
```diff
@@ -0,0 +1,164 @@
...
+  - **Organization, Admin, or Editor permissions**, which are required to create and manage alerts
+
+### Support
+
+If you need help with this integration, contact Dagster Cloud support.
```
Comment (johannkm) [inline docs/content/dagster-cloud/managing-deployments/alerts/pagerduty.mdx:44]: could link https://dagster.io/contact

File: docs/next/public/images/dagster-cloud/alerts/pagerduty-integration-key.png
Comment (johannkm) [inline docs/next/public/images/dagster-cloud/alerts/pagerduty-integration-key.png:1]: micro nit: could filter out the scrollbar

File: docs/content/dagster-cloud/managing-deployments/alerts/pagerduty.mdx [hunk 7]
```diff
@@ -0,0 +1,164 @@
...
+Dagster Cloud supports [sending alerts](/dagster-cloud/managing-deployments/alerts) to PagerDuty, which will automatically create a PagerDuty incident.
+
+---
+
+## PagerDuty + Dagster Cloud integration benefits
+
+- Empower your team with real-time incident response by integrating Dagster Cloud and PagerDuty
+- Notify on-call responders when alerts are triggered in Dagster Cloud
+- Dramatically reduce downtime for issues related to asset materialization, asset checks, job runs, schedules, sensors, and code locations
+- Customize the urgency and routing of PagerDuty incidents based on the content of the Dagster Cloud event payload
```
Comment (johannkm) [inline docs/content/dagster-cloud/managing-deployments/alerts/pagerduty.mdx:23]: I'm not positive we'll support this yet- could we remove

**Review by johannkm — APPROVED** [review]
Looks good! Note the considerations for landing unlisted vs waiting to land

File: docs/content/dagster-cloud/managing-deployments/alerts/pagerduty.mdx (hunk 7 above)
Comment (johannkm) [reply docs/content/dagster-cloud/managing-deployments/alerts/pagerduty.mdx:23]: I think this is the relevant feature https://support.pagerduty.com/docs/event-orchestration#advanced-event-orchestration, I haven't gotten to play around with it yet

File: docs/content/dagster-cloud/managing-deployments/alerts/pagerduty.mdx (hunk 3 above)
Comment (johannkm) [reply docs/content/dagster-cloud/managing-deployments/alerts/pagerduty.mdx:115]: We don't support an env var currently- on my list

File: docs/content/dagster-cloud/managing-deployments/alerts/pagerduty.mdx (hunk 2 above)
Comment (johannkm) [reply docs/content/dagster-cloud/managing-deployments/alerts/pagerduty.mdx]: yep this is good

File: docs/content/_navigation.json (hunk 4 above)
Comment (erinkcochran87) [reply docs/content/_navigation.json:613]: I'll take it out of the nav + category pages for now

File: docs/content/dagster-cloud/managing-deployments/alerts/pagerduty.mdx (hunk 6 above)
Comment (erinkcochran87) [reply docs/content/dagster-cloud/managing-deployments/alerts/pagerduty.mdx:44]: Sounds good - I wasn't sure where to link, tbh

File: docs/content/dagster-cloud/managing-deployments/alerts/pagerduty.mdx (hunk 5 above)
Comment (johannkm) [reply docs/content/dagster-cloud/managing-deployments/alerts/pagerduty.mdx:39]: that this is enterprise only

Comment (johannkm) [reply docs/content/dagster-cloud/managing-deployments/alerts/pagerduty.mdx:39]: probably fine here, but I was thinking something at the top of the page. Not critical

Comment (erinkcochran87) [reply docs/content/dagster-cloud/managing-deployments/alerts/pagerduty.mdx:39]: I could add it to the callout for Dagster Cloud at the top of the page. Something like "This guide is applicable to Dagster Cloud Pro plans."

Comment (johannkm) [reply docs/content/dagster-cloud/managing-deployments/alerts/pagerduty.mdx:39]: perfect

**Pull request merged**